x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/export.cy.ts

```diff
@@ -20,7 +20,6 @@ import { TIMELINE_CHECKBOX } from '../../../screens/timelines';
 import { createTimeline } from '../../../tasks/api_calls/timelines';
 import { expectedExportedTimeline, getTimeline } from '../../../objects/timeline';
 
-// FLAKY: https://github.com/elastic/kibana/issues/165744
```

Comment:
this ticket is actually still open

Reply:
removed per https://github.com/elastic/kibana/pull/173384